tests/preprocessing/test_triplegs.py

```diff
@@ -53,14 +55,34 @@ def test_generate_trips_dtype_consistent(self):
         pfs, tpls = pfs.as_positionfixes.generate_triplegs(stps)
         
         # generate trips and a joint staypoint/triplegs dataframe
-        stps, tpls, trips = ti.preprocessing.triplegs.generate_trips(stps, 
-                                                                     tpls, 
+        stps, tpls, trips = ti.preprocessing.triplegs.generate_trips(stps, tpls,
                                                                      gap_threshold=15, 
                                                                      id_offset=0)
         
         assert stps['user_id'].dtype == trips['user_id'].dtype
         assert trips.index.dtype == "int64"
         
+        assert stps['trip_id'].dtype == "float"
```

Comment:
I think this is what I mean in #131. Here we have the index as int64 and the references to that index as float...

Reply:
I had a look at the issue and I agree we could change into `Int64`. Now (I think) the framework is complete and all we need is to change `float` into `Int64` in generate functions and the dtype_consistent tests.

Reply:
You can notice that the framework gets more _complete_ because the pull requests are getting easier :D